Source/WebCore/css/parser/CSSSelectorParser.cpp

```diff
@@ -287,7 +287,9 @@ static OptionSet<CompoundSelectorFlag> extractCompoundFlags(const MutableCSSSele
     // FIXME: https://bugs.webkit.org/show_bug.cgi?id=161747
     // The UASheetMode check is a work-around to allow this selector in mediaControls(New).css:
     // input[type="range" i]::-webkit-media-slider-container > div {
-    if (parserMode == UASheetMode && simpleSelector.pseudoElement() == CSSSelector::PseudoElement::UserAgentPart)
+    if (parserMode == UASheetMode
+        && (simpleSelector.pseudoElement() == CSSSelector::PseudoElement::UserAgentPart
+        || simpleSelector.pseudoElement() == CSSSelector::PseudoElement::WebKitUnknown))
```

Comment:
Given we control the UA stylesheet, I don't expect us to have unknown -webkit- pseudo elements in there. We should also remove this special case.